modules/querier/querier.go

```diff
@@ -92,13 +91,14 @@ func New(cfg Config, clientCfg ingester_client.Config, ring ring.ReadRing, store
 }
 
 func (q *Querier) CreateAndRegisterWorker(tracesHandler http.Handler) error {
-	worker, err := cortex_frontend.NewWorker(
+	worker, err := cortex_worker.NewQuerierWorker(
 		q.cfg.Worker,
-		cortex_querier.Config{
+		/*cortex_worker.Config{
```

Comment:
Is the worker supplied with `MaxConcurrentQueries` in another way?

Reply:
This is updated, see latest.